pkgs/top-level/all-packages.nix

```
@@ -20823,6 +20823,12 @@ in
     lcms = lcms2;
   };
 
+  inkscape-with-extensions = callPackage ../applications/graphics/inkscape/with-extensions.nix {
+    lndir = xorg.lndir;
+  };
+
+  inkscape-extensions = callPackage ../applications/graphics/inkscape/extensions.nix {};
```

since this file evaluates as a set of derivations:
```suggestion
  inkscape-extensions = callPackages ../applications/graphics/inkscape/extensions.nix {};
```